homeassistant/components/plex/media_player.py

```diff
@@ -491,14 +491,22 @@ def play_media(self, media_type, media_id, **kwargs):
                 ) from err
         else:
             shuffle = src.pop("shuffle", 0)
+            continuous = src.pop("continuous", 0)
```

Comment:
Looks like `services.yaml` needs to be updated ?

Reply:
Since these are all part of `media_player.play_media` I _think_ the convention I've seen is to not include these special parameters in `services.yaml` and to instead rely on the integration's documentation page.

Reply:
I understand the `services.yaml` to only apply to the current integration's domain (`plex` in this case). 

No other integrations provide a description for the `media_player` platform service call:
```
$ grep play_media homeassistant/components/*/services.yaml
homeassistant/components/media_extractor/services.yaml:play_media:
homeassistant/components/media_player/services.yaml:play_media:
```
This should be addressed by updating documentation on the Plex page: https://www.home-assistant.io/integrations/plex/.